src/svcutilcore/src/Microsoft/Tools/ServiceModel/SvcUtil/CommandLineParser.cs

```diff
@@ -57,21 +58,19 @@ internal SwitchType SwitchType
         }
 
         internal bool Equals(string other)
-        {
-            string temp;
-
-            //ensure that compare doesn't start with '/' or '-'
+        { 
+            //ensure that compare doesn't start with '--' or '-'
             //also convert to lower-case
-            if ((other[0] == '/') || (other[0] == '-'))
-                temp = (other.Substring(1)).ToLower(CultureInfo.InvariantCulture);
-            else
-                temp = other.ToLower(CultureInfo.InvariantCulture);
-
+            if(other.StartsWith("--"))
+                other = other.Substring(2);
```

Comment:
Nit, formatting. There should be a space between the if and the open parenthesis. Same on the else if.